storage/cloud-client/snippets_test.py

```diff
@@ -287,12 +288,29 @@ def test_rename_blob(test_blob):
     except google.cloud.exceptions.exceptions.NotFound:
         pass
 
-    storage_move_file.rename_blob(bucket.name, test_blob.name, "test_rename_blob")
+    storage_rename_file.rename_blob(bucket.name, test_blob.name, "test_rename_blob")
 
     assert bucket.get_blob("test_rename_blob") is not None
     assert bucket.get_blob(test_blob.name) is None
 
 
+def test_move_blob(test_bucket_create, test_blob):
+    bucket = test_blob.bucket
+    storage.Client().create_bucket(test_bucket_create)
+
+    try:
+        test_bucket_create.delete_blob("test_move_blob")
+    except google.cloud.exceptions.NotFound:
+        pass
```

Comment:
```suggestion
        print("Blob already deleted, likely in the test")
```

Avoid putting `pass` in exceptions - we prefer adding a test statement to help when debugging

Reply:
feel free to ping when you're ready for rereview

Reply:
Thanks @leahecole! Added the print statements. Ready for review 🙂